src/util/createStore.js

```diff
@@ -5,9 +5,19 @@
 import { applyMiddleware, createStore, compose } from 'redux';
 import getPlatformMiddleware from '../middleware/epic';
 import { fetchQueries, mergeCookies } from '../util/fetchUtils';
+import getClickTracker from './clickTracking';
 
 const noopMiddleware = store => next => action => next(action);
 
+export const clickTrackEnhancer = createStore => (reducer, initialState, enhancer) => {
+	const store = createStore(reducer, initialState, enhancer);
+	const clickTracker = getClickTracker(store);
+	document.body.addEventListener('click', clickTracker);
+	document.body.addEventListener('change', clickTracker);
```

Comment:
This is copied from what is happening in Meetup Classic, and it looks like the `change` listener was added only in order to pick up change events on `<select>` elements with a `data-clicktracking` attribute - there's a check at the top of the `clickTracker` function that bails for `change` events on elements that don't have the attribute.

I'm not sure what product motivations went into that, but I think you're right that those elements would get some redundant `click` data in addition to the `change` data when the user was using a mouse. I'm reluctant to 'fix' that here, though, without a better defined specification of what is actually needed.